### PR TITLE
[stable/fluent-bit] Upgrade to Fluent Bit v1.1.0

### PR DESCRIPTION
**Pull request by edsiper**
#### What this PR does / why we need it:

Upgrade to the latest stable version of Fluent Bit v1.1.0.

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/fluent-bit/values.yaml [hunk 1]
```diff
@@ -5,8 +5,8 @@ on_minikube: false
 image:
   fluent_bit:
     repository: fluent/fluent-bit
-    tag: 1.0.6
-  pullPolicy: IfNotPresent
+    tag: 1.1.0
+  pullPolicy: Always
```
Comment (mtparet) [inline stable/fluent-bit/values.yaml:9]: Is the always policy needed by default ?

Comment (edsiper) [reply stable/fluent-bit/values.yaml:9]: Ideally yes (maintainers stats), this was changed by mistake in a previous version, user can always adjust it as required .

Comment (mtparet) [reply stable/fluent-bit/values.yaml:9]: Ok, thanks for the answer!

**Comment by edsiper** [comment]
cc: @kfox1111

File: stable/fluent-bit/values.yaml (hunk 1 above)
Comment (kfox1111) [inline stable/fluent-bit/values.yaml:9]: Why change the pull policy? The image shouldn't ever change.

Comment (edsiper) [reply stable/fluent-bit/values.yaml:9]: I am reverting to the original value we had, Pull Policy was changed by 00a84538d7 in January this year without noticing it. 

From a maintainer perspective that's' our only metric to get a notion of the number of deployments. Users can still change to their preferred value.

Comment (kfox1111) [reply stable/fluent-bit/values.yaml:9]: hmm. ok. Makes sense.

Comment (edsiper) [reply stable/fluent-bit/values.yaml:9]: @kfox1111 can you help to review the new property of Kube_Tag_Prefix ?

Comment (kfox1111) [reply stable/fluent-bit/values.yaml:9]: hmm... so there is:
 Match               {{ .Values.filter.kubeTag }}.*
Kube_Tag_Prefix     {{ .Values.filter.kubeTag }}.

Both set to the same values, with one with a star and the other with just a dot. is that right? Do we need them to be separably set?

Comment (kfox1111) [reply stable/fluent-bit/values.yaml:9]: kubeTag is set by default to 'kube'. but the docs for Kube_Tag_prefix show something like: kube.var.log.containers.

Maybe kubeTagPrefix should be its own value so its independently set able.

Comment (edsiper) [reply stable/fluent-bit/values.yaml:9]: actually Kube_Tag_Prefix by default must point to: 

```
{{ .Values.filter.kubeTag }}.var.log.containers.
```
here is an explanation of the changes and config to consider:

https://docs.fluentbit.io/manual/installation/upgrade_notes

Comment (edsiper) [reply stable/fluent-bit/values.yaml:9]: hmm make sense, I will do the change

Comment (edsiper) [reply stable/fluent-bit/values.yaml:9]: updated here: https://github.com/helm/charts/pull/13702/commits/2d7e21935516f8376c5f34dc2dd4ebc796f4c0c6

Comment (edsiper) [reply stable/fluent-bit/values.yaml:9]: @kfox1111 can you test it ?

Comment (kfox1111) [reply stable/fluent-bit/values.yaml:9]: I don't have a quick place to do a test at the moment. sorry.

**Comment by dperetti** [comment]
This updated chart doesn't provide a way to use 1.1 stream processing, does it ?

**Comment by edsiper** [comment]
@dperetti nope, you have to edit the configmap

**Pull request merged**